server/handlers/project_command_output_handler.go

```diff
@@ -29,7 +31,7 @@ type ProjectCommandOutputHandler interface {
 	Clear(ctx models.ProjectCommandContext)
 
 	// Receive will create a channel for projectPullInfo and run a callback function argument when the new channel receives a message.
-	Receive(projectInfo string, callback func(msg string) error) error
+	Receive(projectInfo string, ch chan string, callback func(msg string) error) error
```

Comment:
You can probably remove argument names from the interface. Or update the `ch` to `receiver`

Reply:
Done!